ci/check-basics.py

```diff
@@ -79,6 +79,28 @@ def complain(message):
 
         mode = raw_diff.split()[1]
 
+        ABSOLUTE_PATH = OMZ_ROOT / path
```

Comment:
```suggestion
        absolute_path = OMZ_ROOT / path
```

(this is not a constant)

Reply:
Renamed